scripts/eqtl_hail_batch/generate_eqtl_spearman.py

```diff
@@ -656,7 +661,8 @@ def main(
     copy_common_env(filter_mt_job)
     filtered_mt_path = filter_mt_job.call(prepare_genotype_info, keys_path=keys)
 
-    for idx in range(n_genes_in_scatter):
+    # for idx in range(n_genes_in_scatter):
```

Comment:
Should this be reverted?

Reply:
yes, good catch, thanks! I'll update that